cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/InboundNetworkCapacity.java

```diff
@@ -43,14 +43,24 @@ protected InboundNetworkCapacity(BrokerCapacity brokerCapacity, BrokerCapacityOv
      *
      * @return The capacity of resource represented as a String.
      */
-    private static String processResourceCapacity(BrokerCapacity brokerCapacity,
-                                          BrokerCapacityOverride brokerCapacityOverride) {
+    private String processResourceCapacity(BrokerCapacity brokerCapacity,
+                                           BrokerCapacityOverride brokerCapacityOverride) {
+
         if (brokerCapacityOverride != null && brokerCapacityOverride.getInboundNetwork() != null) {
+
+            this.isUserConfigured = true;
             return brokerCapacityOverride.getInboundNetwork();
+
         } else if (brokerCapacity != null && brokerCapacity.getInboundNetwork() != null) {
+
+            this.isUserConfigured = true;
             return brokerCapacity.getInboundNetwork();
+
         } else {
+
+            this.isUserConfigured = false;
             return DEFAULT_NETWORK_CAPACITY_IN_KIB_PER_SECOND;
+
         }
```

Comment:
No need for the empty lines here, it is just a simple if-ifelse-if:

```suggestion
    private String processResourceCapacity(BrokerCapacity brokerCapacity,
                                           BrokerCapacityOverride brokerCapacityOverride) {
        if (brokerCapacityOverride != null && brokerCapacityOverride.getInboundNetwork() != null) {
            this.isUserConfigured = true;
            return brokerCapacityOverride.getInboundNetwork();
        } else if (brokerCapacity != null && brokerCapacity.getInboundNetwork() != null) {
            this.isUserConfigured = true;
            return brokerCapacity.getInboundNetwork();
        } else {
            this.isUserConfigured = false;
            return DEFAULT_NETWORK_CAPACITY_IN_KIB_PER_SECOND;
        }
```

Reply:
Fixed.